src/_pytest/stepwise.py

```diff
@@ -8,53 +9,70 @@
 from _pytest.main import Session
 from _pytest.reports import TestReport
 
+if TYPE_CHECKING:
+    from _pytest.cacheprovider import Cache
+
+STEPWISE_CACHE_DIR = "cache/stepwise"
+NO_PREVIOUSLY_FAILED_WONT_SKIP = "no previously failed tests, not skipping."
+PREVIOUSLY_FAILED_TEST_NOT_FOUND = "previously failed test not found, not skipping."
+SKIPPING_PASSED_ITEMS = "skipping {} already passed items."
+
 
 def pytest_addoption(parser: Parser) -> None:
     group = parser.getgroup("general")
     group.addoption(
         "--sw",
         "--stepwise",
         action="store_true",
+        default=False,
         dest="stepwise",
         help="exit on test failure and continue from last failing test next time",
     )
     group.addoption(
         "--stepwise-skip",
+        "--sw-skip",
```

Comment:
For consistency with `--sw`:

```suggestion
        "--sw-skip",
        "--stepwise-skip",
```